test/edgehog_device_forwarder/websockets_test.exs

```diff
@@ -0,0 +1,134 @@
+# Copyright 2023 SECO Mind Srl
+# SPDX-License-Identifier: Apache-2.0
+
+defmodule EdgehogDeviceForwarder.WebSocketsTest do
+  use EdgehogDeviceForwarder.CacheCase
+  use EdgehogDeviceForwarder.MessageCase
+  alias EdgehogDeviceForwarderProto.Edgehog.Device.Forwarder.Message, as: ProtoMessage
+  alias EdgehogDeviceForwarderProto.Edgehog.Device.Forwarder.WebSocket, as: ProtoWebSocket
+  alias EdgehogDeviceForwarder.{HTTPRequests, WebSockets}
+  alias EdgehogDeviceForwarder.WebSockets.Data
+
+  require Logger
+
+  setup_all do
+    [valid_message: {:text, "1"}]
```

Comment:
This value is constant, so it could be a module attribute rather than a setup_all step

Reply:
done